operator/cmd/mesh/operator-init.go

```diff
@@ -165,6 +165,13 @@ func applyManifest(manifestStr, componentName string, opts *kubectlcmd.Options,
 	opts.Prune = pointer.BoolPtr(false)
 	out, objs := manifest.ApplyManifest(name.ComponentName(componentName), manifestStr, version.OperatorBinaryVersion.String(), *opts)
 
+	if opts.Wait {
+		err := manifest.WaitForResources(objs, opts)
```

Comment:
Is there a default waittimeout number exists?

Reply:
Yes, default values is 300 seconds.